docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -0,0 +1,163 @@
+the [Miniforge3-MacOSX-arm64.sh](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh)
+installation script, and then do the following:
+~~~
+# Install miniforge. Accept all default choices
+bash Miniforge3-MacOSX-arm64.sh
+# Activate conda base environment for current session
+eval "$(/Users/gwpacworkstation/miniforge3/bin/conda shell.zsh hook)"
+# Activate conda at startup (to make sure you're always using native python)
+conda init zsh
+exit # close shell. conda will automatically load when new zsh shells start.
+~~~
+
+Open a new terminal window. Then, install the necessary python packages. (Note:
+jupyter and notebook are recommended, not required.)
+~~~
+conda install numpy scipy matplotlib h5py pyyaml jupyter notebook
+~~~
+
+### 3. Install dependencies with Homebrew
+
+Most of spectre's dependencies beyond python can be installed using the
+[homebrew](https://brew.sh) package manager. First, install Homebrew by
+following the instructions on the [homebrew](https://brew.sh) homepage. Then,
+run the following to install a fortran compiler and other dependencies:
+~~~
+brew install gcc
+brew install openblas boost gsl cmake doxygen
+brew install ccache autoconf automake catch2 jemalloc hdf5 pybind11 yaml-cpp
+~~~
+
+### 4. Install remaining dependencies
+
+Here, install the remaining dependencies that cannot be installed
+with homebrew or miniforge.
+
+Create a directory (such as `$HOME/apps`) to store the build dependencies.
+~~~
+export SPECTRE_DEPS_ROOT=$HOME/apps
+~~~
+
+~~~
+cd $SPECTRE_DEPS_ROOT
+mkdir blaze
+pushd blaze
+curl -L https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.8.tar.gz \
+> blaze-3.8.tar.gz
+tar -xf blaze-3.8.tar.gz
+mv blaze-3.8 include
+popd
+
+git clone https://github.com/edouarda/brigand.git
+
+# Need master branch of libxsmm to support Apple Silicon
+git clone https://github.com/hfp/libxsmm.git
+pushd libxsmm
+make
+popd
+
+pushd ./src
+git clone https://github.com/Libsharp/libsharp.git
+cd Libsharp
+
+# Do not use compiler flag -march=native (unsupported on Apple Silicon)
+sed "s/-march=native//" configure.ac > configure.ac.mod
+mv configure.ac.mod configure.ac
+
+autoupdate
+autoconf
+./configure
+make
+mv auto $SPECTRE_DEPS_ROOT/libsharp
+popd
+~~~
+
+Next, install charm++. Note that Apple Silicon Macs require v7.0.0.
+
+~~~
+git clone https://github.com/UIUC-PPL/charm
+pushd charm
+git checkout v7.0.0
+./build charm++ multicore-darwin-arm8 --with-production -g3 -j
+popd
+~~~
+
+### 5. Clone, configure, and build SpECTRE.
+Next, clone SpECTRE, make a build directory, and configure. In whatever
+directory you prefer, clone SpECTRE and make a build directory, e.g.
+~~~
+git clone git@github.com:sxs-collaboration/spectre.git
+cd spectre
+mkdir build
+cd build
+~~~
+
+Next, configure SpECTRE using the following `cmake command`.
+
+~~~
+cmake -D CMAKE_C_COMPILER=clang -D CMAKE_CXX_COMPILER=clang++ -D \
+CMAKE_Fortran_COMPILER=gfortran -D BUILD_PYTHON_BINDINGS=OFF \
```

Comment:
I was able to build the python bindings

Reply:
I was as well, so I updated the docs to suggest turning python bindings on